compiler/src/main/java/org/qbicc/type/CompoundType.java

```diff
@@ -256,6 +257,10 @@ public Builder setOverallAlignment(int align) {
             return this;
         }
 
+        public Builder addNextMember(final ValueType type) {
+            return addNextMember("", type, type.getAlign());
```

Comment:
Should this only be applicable to INLINE tagged CompoundTypes?  This is not a leading question - I don't know whether that's a useful restriction or not.

Reply:
I think it makes sense to restrict this to INLINE only. Otherwise getMember(String name) may not retrieve the expected result. I will create a new pr for that.

Reply:
Here is the change: https://github.com/qbicc/qbicc/pull/624